### DISCUSSION TITLE
Bug on many items during spawn

### DISCUSSION THREAD

**Discussion by Theo0307** [MAFINS/MenyooSP#589] [General]
Hello, 

I've been using your menu mode on GTA V in story mode for a while now, and I'm having a lot of problems when I want to spawn vehicles, for example. 
When I spawn a 10F for example, I can drive it for 10 seconds and then it disappears. It's the same with many planes and helicopters.

I'd like to know what's causing this, if it's a bad installation on my part or if the script has a bug.

**Comment by metoxys** [comment]
Duplicate of #579  
Consider using the solution https://github.com/MAFINS/MenyooSP/issues/579#issuecomment-1597341536
